packages/parser/src/util.ts

```diff
@@ -37,7 +38,7 @@ export function normalizeArguments<S extends APIDocument = APIDocument>(
  *
  */
 export function convertOptionsForParser(options: ParserOptions): Partial<$RefParserOptions> {
-  return getNewOptions({
+  return merge(getJsonSchemaRefParserDefaultOptions(), {
```

Comment:
Refactored this to not use the `getNewOptions` method that they have in their dist that essentially just invokes a custom `merge()` method. I tried my best to _not_ do this with lodash but ran into a whole bunch of weird issues. Might take another stab in a followup because I would really love to remove lodash from this.